src/community/propose-a-component-or-pattern/index.md.njk

```diff
@@ -10,7 +10,8 @@ Anyone can propose a new component or pattern for the GOV.UK Design System.
 
 To be successful, proposals need to show that the component or pattern being suggested would be [useful and unique](/community/contribution-criteria).
 
-Follow 3 steps to propose a component or pattern for the Design System.
+- Follow 3 steps to propose a component or pattern for the Design System.
```

Comment:
Hey @digitalcourtney87!

Thanks for this change it looks good but there's one thing that needs fixing before we can publish this.

The - and + characters are to represent what needs to be added can be removed.

You can edit the file again by using the following link: https://github.com/digitalcourtney87/govuk-design-system/edit/patch-1/src/community/propose-a-component-or-pattern/index.md.njk?pr=%2Falphagov%2Fgovuk-design-system%2Fpull%2F767

Let us know if you need any help with this change.

Reply:
Thanks @nickcolley I've updated!